### DISCUSSION TITLE
dashboard 24h history ?

### DISCUSSION THREAD

**Discussion by Tobi7241** [beep-projects/SystaPi#22] [General]
when I load http://systapi:1337/SystaREST/dashboard the diagrams are empty and does not show past data.
when looking in the browser DevTools, the REST API is only looking for actual data (/SystaREST/servicestatus and /SystaREST/status). How can I see historical data ?

The diagram x-axis show 24 hours but the data received are not mapped to the right x-axis.

initial loading:
![grafik](https://github.com/beep-projects/SystaPi/assets/62337620/3b2321be-1ce0-46b6-970d-6b7390fb88e7)
after 1 minute:
![grafik](https://github.com/beep-projects/SystaPi/assets/62337620/905c9db1-1025-48b7-bef3-a27ccfb156af)

**Comment by beep-projects** [comment]
Hey Tobi, 

SystaPi is designed to provide a REST endpoint for accessing Systa Comfort data. It does not include a database as everyone with a home automation system will have their own system installed.

The dashboard is only meant as a small demo to show how the data could be used. Someone would have to add a storage, at the moment everything is only volatile and after a restart the plots are empty again.
